src/openeo_aggregator/backend.py

```diff
@@ -160,18 +172,58 @@ def _merge_collection_metadata(self, by_backend: Dict[str, dict]) -> dict:
                             or [[None, None]],
             },
         }
-        result["links"] = list(getter.union("links"))
+        # links
+        result["links"] = [l for l in list(getter.union("links")) if l["rel"] not in ("self", "parent", "root")]
+        result["links"].append({
```

Comment:
Ultimately, I think we should do these self/parent/root links for all collections, not only the ones with merging

Reply:
Added _normalize_metadata method to AggregatorCollectionCatalog.